examples/adjoints/adjoints_ex5/adjoints_ex5.C

```diff
@@ -502,7 +502,7 @@ int main (int argc, char ** argv)
 
           // Output the H1 norm of the computed solution
           libMesh::out << "|U("
-                      << system.time + system.time_solver->last_complete_deltat()
+                      << system.time + system.time_solver->last_completed_deltat()
```

Comment:
I like the new name better too, but is it worth API breakage?  I guess we haven't had the old name around very long?  But if it's worth changing then it's worth keeping a libmesh_deprecated() shim for the old name for a while.

Reply:
The only other place we have used this is adjoints example 5, and I updated that. We can make a shim, but I doubt anyone is using this.